BWAPI4J/src/main/java/bwem/map/MapInitializerImpl.java

```diff
@@ -155,7 +155,13 @@ public void initialize(final boolean enableTimer) {
       logger.info("Map::createBases: " + timer.elapsedMilliseconds() + " ms");
       timer.reset();
     }
-
+    
+    assignStartingLocationsToSuitableBases();
```

Comment:
This function will throw an exception if it fails to match suitable base locations to all starting locations. Lots of maps play around with start locations and observer player starting locations. So, this change breaks more than it fixes, IMO. If the intent of this change is so the user is not required to manually invoke this function during the bot's initialization stages, the thrown exception needs to be removed or dealt with. As something to consider, BWEM generates a list of starting locations not assigned to bases if the function fails.

Reply:
Understood, I'll withdraw this proposal. Was just struck by how user-unfriendly it was to have an `initialize()` then have to manually do something that seems like core functionality, not an option.